pkg/vm/vm.go

```diff
@@ -3,6 +3,7 @@ package vm
 import (
 	"fmt"
 
+	assembler "github.com/NethermindEth/cairo-vm-go/pkg/assembler"
```

Comment:
`assembler` is a very long name, what do you think of using a shortest one, like `a` or `as`?